systems/controllers/QPCommon.cpp

```diff
@@ -272,6 +338,37 @@ std::vector<SupportStateElement> loadAvailableSupports(std::shared_ptr<drake::lc
   return available_supports;
 }
 
+void addJointSoftLimits(const JointSoftLimitParams &params, const DrakeRobotState &robot_state, const VectorXd &q_des, std::vector<SupportStateElement> &supports, std::vector<drake::lcmt_joint_pd_override> &joint_pd_override) {
+  Matrix<bool, Dynamic, 1> has_joint_override = Matrix<bool, Dynamic, 1>::Zero(q_des.size());
+  for (std::vector<drake::lcmt_joint_pd_override>::iterator it = joint_pd_override.begin(); it != joint_pd_override.end(); ++it) {
+    has_joint_override(it->position_ind - 1) = 1;
+  }
+  for (int i=0; i < params.lb.size(); i++) {
+    if (!has_joint_override(i) && params.enabled(i)) {
+      int disable_body_1idx = params.disable_when_body_in_support(i);
+      if (disable_body_1idx == 0 || !inSupport(supports, disable_body_1idx - 1)) {
+        double w_lb = 0;
+        double w_ub = 0;
+        if (!isinf(params.lb(i))) {
```

Comment:
isinf is not available in MSVC2010. use isInf from drakeFloatingPointUtil.h instead.